### PR TITLE
Fixed System.xml and add Default Values for the missing Configuration

### PR DESCRIPTION
**Pull request by marcbeyond**

### CODE REVIEW

File: Service/CustomerCreator.php [hunk 1]
```diff
@@ -131,16 +131,21 @@ public function prepareCustomers($customerGroup)
     public function prepareEmail($customerGroupId)
     {
         $domainPrefix = $this->getConfig()['domain'];
+        if(!$domainPrefix){
+            $domainPrefix = $_SERVER['SERVER_NAME'];
```
Comment (krisdante) [inline Service/CustomerCreator.php:135]: This is not how it shall work. We actually need this setting to be added. And it shall be "faked" domain - hard to guess, not the pure shop domain. 
We set this in settings during provisioning

**Review by krisdante — COMMENTED** [review]
Thank you for this contribution.
We will however not merge it for now. 
It is our decision by design to not have those options in Admin Area.
Changing those settings by Admin can lead to unexpected effects.

We populate those settings directly to Database during provisioning (deployment).
You can see this in our deployment stack:
Role:
https://github.com/mageops/ansible-infrastructure/blob/master/roles/cs.magento-configure/tasks/080-configure-integrations.yml

Settings:
https://github.com/mageops/ansible-infrastructure/blob/master/group_vars/all.yml#L820

We do it this way to be able to re-use those settings for Supervisord (also to be checked in MageOps scripts).

**Comment by krisdante** [comment]
If there is a need to tune those settings in admin - There has to be an additional checkbox where Admin confirms that is aware of dangerous side effects. Also changing these settings after users for crawlers are generated will be seriously bad wrong.

**Comment by Alex7Lem** [comment]
After installing and removing the page-cache-warmer modules, errors occur when opening pages in the admin panel:
Catalog / Products / Action / Edit
Catalog / Categories

Installation:
composer require creativestyle/magesuite-page-cache-warmer

Deleting:
composer remove creativestyle/magesuite-page-cache-warmer creativestyle/magesuite-page-cache-warmer-crawler creativestyle/magesuite-page-cache-warmer-crawl-worker

Magento ver. 2.3.5-p1

**Pull request closed without merging**